src/types/CompletedStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum CompletedStatus {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
```

Comment:
I think its why your tests of filtering are failing. Actually, the main values start from the small characters. Yours starts with capitals

Reply:
done